index.js

```diff
@@ -1,10 +1,14 @@
-#!/usr/bin/env node
-
 const fetch = require('node-fetch');
 const fs = require('fs');
 const assert = require('assert');
 
 const { jsonDiff } = require('./diff');
+const noop = () => {}
+
+let logger = {
```

Comment:
I see why you added this, to suppress logging when using `combineSpecs` in your node.js scripts but this is an unreliable way to achieve this.
What if I use `run` and then `combineSpecs` for some reason in my code?

I think better to add `silent` option to both functions or configure it using an env variable.